### DISCUSSION TITLE
Use Antd with cdn links

### DISCUSSION THREAD

**Discussion by vmkBB** [ant-design/ant-design#40738] [Q&A]
How to use antd library with cdn links(without using npm) and extract the components  and use them.

**Comment by yoyo837** [comment]
https://ant.design/docs/react/introduce#import-in-browser

**Reply by vmkBB** [reply]
I added the cdn links and now how to access the antd components in react application. If possible please share the code snippet.

**Reply by ink-ru** [reply]
You have to use require instead inport:
`const AntD = require('antd');`
so component call will look like:
`<AntD.DatePicker />`
or require each component separatly:
`const Button = require('antd/lib/button');`

See more: [https://stackoverflow.com/questions/59406271/](https://stackoverflow.com/questions/59406271/is-it-possible-to-use-ant-design-straight-into-the-browser-in-combination-with-r)

**Comment by voltuer** [comment]
this is not working, please udpate the documentation because i keep getting `ReferenceError: antd is not defined`

**Comment by afc163** [comment]
6.x ❌：https://codepen.io/afc163/pen/zxBjOpv?editors=001
5.x ✅：https://codepen.io/afc163/pen/yyJjBPd?editors=001
